source/extensions/extensions_metadata.yaml

```diff
@@ -539,6 +539,16 @@ envoy.quic.proof_source.filter_chain:
   - envoy.quic.proof_source
   security_posture: robust_to_untrusted_downstream
   status: alpha
+envoy.quic.udp_packet_writer.default:
```

Comment:
hm, I buy these can be in line with other quic extensions but I think we were supposed to bump them out of alpha when QUIC went GA.  mind following a tracking bug for that?

Reply:
Good point. Filed https://github.com/envoyproxy/envoy/issues/21377